### PR TITLE
Rename variable to MicrosoftNETCoreAppRuntimeVersion

### PR DESCRIPTION
**Pull request by natemcmaster**
No functional change, but I think this naming is probably better.

### CODE REVIEW

**Comment by natemcmaster** [comment]
Note: the naming here intentionally "MicrosoftNETCoreAppRuntimeVersion", which does not use the word "package". The plan for 3.0 RMT is to remove Microsoft.NETCore.App. Since the runtime is no longer represented by a simple NuGet package, I was hoping to stop using this concept in our naming and code.

cc @sebastienros @jkotalik

File: eng/Versions.props [hunk 1]
```diff
@@ -48,6 +44,13 @@
     <!-- Packages from dotnet/arcade -->
     <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.19309.1</MicrosoftDotNetGenAPIPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
```
Comment (jkotalik) [inline eng/Versions.props]:
```suggestion
  <PropertyGroup Label="Manual" >
```

Comment (jkotalik) [reply eng/Versions.props]: Oops, I see now. Why not just put this version in the manual versions?

Comment (natemcmaster) [reply eng/Versions.props]: It's not really a manual package version either. Maybe 'Label="Dependency version settings"'?

Comment (jkotalik) [reply eng/Versions.props]: Sure. It's super nitpicky anyways. I just want us to have the same changes.

Comment (natemcmaster) [reply eng/Versions.props]: Ok, updated to add a label.

**Pull request merged**